eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -351,8 +351,8 @@ object Graph {
     val BLOCK_TIME_TWO_MONTHS = 8640
 
     // Low/High bound for channel capacity
-    val CAPACITY_CHANNEL_LOW = (1000 sat).toMilliSatoshi
-    val CAPACITY_CHANNEL_HIGH = Channel.MAX_FUNDING.toMilliSatoshi
+    val CAPACITY_CHANNEL_LOW = Btc(0.001).toMilliSatoshi
```

Comment:
Honestly I wouldn't change this right now.
I'd first wait to see wumbo channels appear in the network, then run path-finding tests on the network DB for this change before merging it.

Reply:
I think it doesn't hurt, the values here are used only to compute the heuristic score to make more informed decision in path finding, whatever value we choose here it never excludes the channel from the result set. There is also the argument that currently we consider a small channel a size of 1000sat which is really tiny and realistically not usable from a routing PoV, it seems to me that small channels in the network are slowly disappearing and updating this value here can help to speed up the process because they will be less likely to be selected.

Reply:
Allright then! But do you have a way of running tests on the real network? Changing the path-finding heuristics should (IMO) always be backed by hard evidence that we're not making things worse for real users...

Reply:
I do :) , i use [this](https://github.com/ACINQ/eclair/blob/graph_fuzzy_testing/eclair-core/src/test/scala/fr/acinq/eclair/router/FuzzyGraphSpec.scala#L88) to load any networkDB and run some path-finding on it, I'll report the results here.

Reply:
Here we go: 

The simulations have been run on a up-to-date mainnet db, with a sample size of 1000 randomly chosen nodes used to run 500 path-finds. The test has been repeated with different amounts to evaluate how the heuristic works with different payment size (recall the new wumbo heuristics slightly privileges larger channel) and with a fixed starting point (ACINQ node). The test are reproducible running `FuzzyGraphSpec` in this [branch](https://github.com/ACINQ/eclair/tree/graph_fuzzy_testing), the results are printed in 3 different files and include the actual route that was found. Note that it's okay for the test to not find a route between the nodes, they have been randomly chosen and there might not be a route connecting them.

| TEST                                                 | routes found(master) | routes found(wumbo) | different routes found |
|------------------------------------------------------|----------------------|---------------------|------------------------|
| 1000sat payment, 2 random nodes       | 242                  | 242                 | 58                     |
| 100000sat payment, 2 random nodes     | 164                  | 164                 | 40                     |
| 1000sat payment, ACINQ -> random node | 443                  | 443                 | 55                     |

There are zero cases where the new heuristics would fail in finding a route that was previously found, where the new heuristics finds a different route that is on average shorter than the one found on master (without the wumbo heuristic)

Reply:
Great thanks, let's keep that change then